box.json.dist

```diff
@@ -1,23 +1,16 @@
 {
-    "alias": "psalm.phar",
-    "base-path": "build/psalm",
-    "main" : "./psalm",
-    "output" : "../../build/psalm.phar",
-    "directories" : [
-        "assets",
-        "vendor"
-    ],
-    "finder": [
-        {
-            "in" : "src"
-        }
-    ],
-    "files" : [
-        "config.xsd"
+    "output" : "build/psalm.phar",
+    "files": [
+        "src/command_functions.php",
+        "src/psalm.php",
```

Comment:
is there any reason why those files are not included in the autoload?

Reply:
They're part of Psalm's command-line startup, and themselves handle autoloading